examples/multiple-receive/multiple_receive.md

```diff
@@ -0,0 +1,7 @@
+# Multiple receive
+
+The multiple receive action allows receiving messages from multiple workers.
```

Comment:
The sentence doesn't effectively differentiate between the use cases of multiple receives and alternate receive. Can we mention how it waits for all the values and constructs the map structure in a short sentence?